src/AI/InfantrySpamAI.java

```diff
@@ -0,0 +1,62 @@
+package AI;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Queue;
+
+import CommandingOfficers.Commander;
+import Engine.GameAction;
+import Terrain.GameMap;
+import Terrain.Location;
+import Terrain.TerrainType;
+import Units.UnitModel;
+
+/**
+ *  Just build tons of Infantry and try to rush the opponent.
```

Comment:
It really appears as if "rushing" part doesn't happen, but maybe that's just me.

Reply:
Not yet. I've just about got it working on my end, but I was thinking I might make it a separate pull request to keep any one from being too huge. Maybe I'll roll it into this one, I dunno.